goalie/go_mesh_seq.py

```diff
@@ -185,17 +185,24 @@ def indicate_errors(
                     f", not type '{type(forms)}'."
                 )
 
-            # Loop over each strongly coupled field
-            for f in self.fields:
-                # Loop over each timestep
-                solutions = self.solutions.extract(layout="field")
-                indicators = self.indicators.extract(layout="field")
-                for j in range(len(solutions[f]["forward"][i])):
-                    # Update fields
+            # Loop over each timestep
+            for j in range(self.time_partition.num_exports_per_subinterval[i] - 1):
+                # Loop over each strongly coupled field
+                for f_idx, f in enumerate(self.fields):
+                    # Transfer solutions associated with the current field f
                     transfer(self.solutions[f][FWD][i][j], u[f])
                     transfer(self.solutions[f][FWD_OLD][i][j], u_[f])
                     transfer(self.solutions[f][ADJ][i][j], u_star[f])
                     transfer(self.solutions[f][ADJ_NEXT][i][j], u_star_next[f])
+                    # In case of multiple solution fields, the field that is solved for
+                    # first used the values of other fields from the previous timestep.
```

Comment:
Do you mean "uses" rather than "used"?

Reply:
I put "used" to try to emphasise that I am referring to get_solver :) I changed to "uses" in 1e88b2b0f303b0904fc286c35663774575a55ebc but I'm still not sure how clear this comment block is in general.